src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ValidateBookmark.cs

```diff
@@ -131,22 +131,18 @@ protected override Manifest PostHandleCore(Manifest manifest)
                             content = $"<a href=\"{link}\">{title}</a>";
                         }
 
-                        string errorCode, dateWarning = string.Empty;
+                        string errorCode, errorMessage;
                         if (internalBookmark)
                         {
                             errorCode = WarningCodes.Build.InvalidInternalBookmark;
-                            dateWarning = " NOTE: This Suggestion will be elevated to a Warning on 8/26/2019. Please fix invalid bookmarks as soon as possible.";
+                            errorMessage = $"Invalid link: '{content}'. The file {linkedToFileSrc} doesn't contain a bookmark named '{bookmark}'.";
                         }
                         else
                         {
                             errorCode = WarningCodes.Build.InvalidExternalBookmark;
+                            errorMessage = $"Illegal link: `{content}` -- missing bookmark. The file {linkedToFileSrc} doesn't contain a bookmark named {bookmark}.";
```

Comment:
Could we use the same `errorMessage` for them? They are same before.

Reply:
@superyyrrzz Updated. They are using same log message like before.